src/gui/askforoauthlogindialog.cpp

```diff
@@ -28,7 +28,7 @@ AskForOAuthLoginDialog::AskForOAuthLoginDialog(AccountPtr accountPtr, QWidget *p
 {
     _ui->setupUi(this);
 
-    _ui->label->setText(tr("The account %1 has been logged out by the server.\n\nPlease use your browser to log in to %2.").arg(accountPtr->displayName(), Theme::instance()->appNameGUI()));
+    _ui->label->setText(tr("The account %1 is currently logged out.\n\nProceed to authenticate using your browser.").arg(accountPtr->displayName()));
```

Comment:
I think some "Please [...]" is missing in that sentence. I guess "Please authenticate using your browser" would do.